src/containers/task-group.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Task from '../components/task';
+
+class TaskGroup extends Component {
+
+  processTaskList(list, givenMonth, givenType) {
```

Comment:
Yes, I agree! I wrote the other methods, realised that I was repeating a chunk of code, and then abstracted that into this method, and then didn't have a name for it. Thanks :D